lambda/lambda_handler.py

```diff
@@ -1,9 +1,44 @@
 """Handler."""
+from typing import Any, Dict, List
+
 from github_usage import process_event
 from logger import LOG
 
+LambdaDict = Dict[str, Any]
```

Comment:
I forgot you can do this, very cool

Reply:
It's not really. The type only exists for mypy and you can't say `isinstance(foo, LambaDict)`. Mypy will catch this error while the definition is in the same file but if the definition is imported from somewhere else it, then it becomes a runtime error.

Reply:
Yeah, I reckon dataclasses is what you want for both compile and runtime use.

Reply:
Done